google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadCallConversion.java

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.remarketing;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.services.CallConversion;
+import com.google.ads.googleads.v3.services.CallConversionResult;
+import com.google.ads.googleads.v3.services.ConversionUploadServiceClient;
+import com.google.ads.googleads.v3.services.UploadCallConversionsResponse;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.DoubleValue;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Imports offline call conversion values for calls related to the ads in your account. To set up a
+ * conversion action, run the {@link
+ * com.google.ads.googleads.examples.remarketing.AddConversionAction} example.
+ */
+public class UploadCallConversion {
+
+  private static class UploadCallConversionParams extends CodeSampleParams {
+    @Parameter(names = ArgumentNames.CUSTOMER_ID)
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.CONVERSION_ACTION_ID)
+    private String conversionActionId;
+
+    @Parameter(names = ArgumentNames.CALLER_ID)
```

Comment:
Any formatting rules to call out here, similar to the nice descriptions you have below for timestamps?